static/app/views/performance/browser/webVitals/components/performanceScoreRing.tsx

```diff
@@ -97,7 +97,7 @@ function PerformanceScoreRing({
           cy={cx}
           color={backgroundColors[index]}
           rotate={rotate}
-          onMouseOver={() => onHoverActions?.[index]()}
+          onMouseOver={() => onHoverActions?.()}
```

Comment:
Why didn't TS catch this? Can we improve the type here?

Reply:
Im actually surprised this wasnt caught by ts either. 
Here's the type of onHoverActions:
```onHoverActions?: () => void```
`index` is a number. I'm not sure why, but typescript doesn't actually complain when you try to `[0]` on a `number`

Reply:
maybe something in our ts settings is letting this happen? or its some kind of js trickery